### PR TITLE
Adds object spinning when thrown and *flip

### PR DESCRIPTION
**Pull request by Iamgoofball**
https://www.youtube.com/watch?v=PGNiXGX2nLU

### CODE REVIEW

**Comment by Iamgoofball** [comment]
okay uh that gif's fucked but you can still see the spin

**Comment by Cheridan** [comment]
:scream:

**Comment by RandomMarine** [comment]
Needs a hotkey so we can #STYLE while we dodge lasers and electrodes.

**Comment by SconesC** [comment]
oh christ

**Comment by vista-** [comment]
This begs for a failure chance.
"Honky the Clown does a majestic backflip and lands face first on the floor!" - fall on the floor.
"You hear a sickening crack coming from your leg" - successful flip and a trip to Medbay.

**Comment by Iamgoofball** [comment]
@vista- muh snowflake

**Comment by Incoming5643** [comment]
need to automatically flip if you manage to move into a laser in the same turn it moves into you and you miss each other

**Comment by Fox-McCloud** [comment]
Fox McCloud grabs Iamgoofball aggressively!

Fox McCloud supplexes Iamgoofball!

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -85,6 +85,12 @@
 				message = "<B>[src]</B> flaps its wings."
 				m_type = 2
 
+		if ("flip")
```
Comment (Giacom) [inline code/modules/mob/living/emote.dm]: Would be cool if monkeys could flip randomly.

**Comment by WJohn** [comment]
Also curious, does object spinning affect bullets/lasers any?

**Comment by Iamgoofball** [comment]
@WJohn they do not effect lasers or bullets.

**Comment by Nienhaus** [comment]
:+1:

**Comment by Iamgoofball** [comment]
goddamn it nienhaus

**Comment by Nienhaus** [comment]
I clicked the wrong button

**Comment by Iamgoofball** [comment]
GODDAMN IT NIENHAUS

**Comment by Nienhaus** [comment]
I'm sorry Goof.

**Pull request merged**